uaclient/cli.py

```diff
@@ -1481,14 +1483,15 @@ def action_collect_logs(args, *, cfg: config.UAConfig):
                 for entitlement in entitlements.ENTITLEMENT_CLASSES
                 if issubclass(entitlement, entitlements.repo.RepoEntitlement)
             ),
-        )
+        ]
 
-        for log in ua_logs:
-            if os.path.isfile(log):
-                log_content = util.load_file(log)
+        # also get default logrotated log files
+        for f in state_files + glob.glob("/var/log/ubuntu-advantage*"):
```

Comment:
We can set the path on the logs in `uaclient.conf`, therefore, I think we should update the logic to read that path directly from the `config` module

Reply:
@lucasmoura `cfg.log_file`, `cfg.timer_log_file`, and `cfg.daemon_log_file` are all included in `state_files`.

Logrotate is hardcoded to `/var/log/ubuntu-advantage*.log`, so I used a similar glob to what we have hardcoded there. 

Aside, this means non-default log files aren't getting rotated, but I think thats uncommon enough to not worry about(?)

The constant `/var/log/ubuntu-advantage` should probably not be hardcoded here though, and in `defaults.py` instead, I can make that change.

Thoughts?

Reply:
Ohh I missed that logrotate was using that path directly, fair enough.
And I agree that this should not be a huge issue at the moment, since it should be a rare situation.
And I also agree we can create the constant on the `defaults.py` module